lib/ABI/FunctionType-revng-c.cpp

```diff
@@ -18,6 +18,8 @@
 #include "revng/ABI/Support.h"
 #include "revng/Model/Binary.h"
 
+static Logger Log("abi-function-type-conversion");
```

Comment:
Nice. Logging == good.
Did this help you in debugging things or did you put this as an act of pure kindness towards the "you" of the future?

Reply:
I had most of this for a long time, only it was using `std::cout`, and was getting stored as a stash which I kept popping-pushing.

This time there were some extensive changes which broke it 😞 - and since I was to fix it anyway, I thought I might as well formalize it properly and push to master so that I don't need to juggle it any more (plus this way it will never break again, since it would have to be fixed on the spot ^^).